mlflow/tracking/utils.py

```diff
@@ -19,7 +19,8 @@
 
 
 _TRACKING_URI_ENV_VAR = "MLFLOW_TRACKING_URI"
-_LOCAL_FS_URI_PREFIX = "file:///"
```

Comment:
I am not sure if this is correct. According to wikipedia (https://en.wikipedia.org/wiki/File_URI_scheme), 
```
file://path (i.e. two slashes, without a hostname) is never correct, but is often used.
```
Looking at the test output. It looks like docker does not like it.

The wikipedia article has an example for valid windows path:

```
file://localhost/c$/WINDOWS/clock.avi
file:///c:/WINDOWS/clock.avi
```
Can we maybe detect windows path and convert them to the form above instead of changing the prefix?

Reply:
Originally I kept the ///, however, it parsed the url as:
>>> urllib.parse.urlparse("file:///c://windows\path")
ParseResult(scheme='file', netloc='', path='/c://windows\\path', params='', query='', fragment='')

I will update to file:///, and parse out the / prefix.

This will keep the code more consistent, self.artifact_dir will also limit the number of times this minor pre-processing has to be done.

Reply:
Makes sense, thanks.